src/features/authentication/forgot_password/forgotPassword.tsx

```diff
@@ -0,0 +1,157 @@
+import React, { useEffect, useState } from "react";
+import { sendPasswordResetEmail } from "firebase/auth";
+import { useNavigate } from "react-router-dom";
+import { InputText } from "primereact/inputtext";
+import { Button } from "primereact/button";
+
+import logo from "../../../common/assets/Logo.png";
+import "./forgotPassword.scss";
+
+import { observer } from "mobx-react-lite";
+import { auth } from "../../../config/firebase-config";
+
+export const ForgotPasswordForm: React.FC = observer(() => {
+  const navigate = useNavigate();
+
+  const [isSubmitPress, setIsSubmitPress] = useState(false);
+  const [successfulSubmit, setSuccessfulSubmit] = useState(false);
+
+  const initialValues = {
+    email: "",
+    password: "",
+  };
+  const [formErrors, setFormErrors] = useState(initialValues);
+  const [email, setEmail] = useState("");
+
+  const forgotPassword = (email: string) => {
+    return sendPasswordResetEmail(auth, email);
+  };
+
+  const validate = (value: any) => {
```

Comment:
Since this function returns an object it would be more clear to name it with a 'get' prefix. Something like `getErrors`.

Reply:
I glanced over the code and changed the local variable: "errors" to "validationErrors" for a better understanding, and "validate()" to "getErrors()". Thanks for the suggestion!